### ISSUE TITLE
a11y: Color Contrast - Banners

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13710]
Insufficient color contrast for standard text

## Actual results

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF
Ratio: 3.16:1 
Main Content
-Text, Accept Button Text, Text to display (on hover and focus) (available when you activate the Show Banner in Footer , Show Consent Banner on Login Screen, Show Login Consent as a modal dialog and Show custom login error checkboxes) 
- Font Size (dropdown on focus) 

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content 
- 10px, 12px, 14px, 16px, 18px, 20px options in dropdown (on hover and on focus) 
- Apply button (on default state) 

Foreground Text Color: #FFFFFF
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Apply button (on focus) 

Foreground Text Color: #B6B6C2
Background Text Color: #FFFFFF
Ratio: 2.00:1 
Main Content 
- #393960
- #eeeff4
- 14px (selected option of the dropdown) 

Foreground Text Color: #FFFFFF 
Background Text Color: #B6B6C2
Ratio: 2.0:1
Main Content
-14px (dropdown option in selected state when keyboard focus is not on option)"

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
>
> Instances:
> 
> Foreground Text Color: #FFFFFF 
> Background Text Color: #B6B6C2
> Ratio: 2.0:1
> Main Content
> -14px (dropdown option in selected state when keyboard focus is not on option)
> 
>
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 233
